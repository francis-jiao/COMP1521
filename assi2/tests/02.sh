# initHeap ... should make heapSize == 4096
./test1 1000
