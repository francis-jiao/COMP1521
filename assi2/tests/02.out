+00000 (F, 4096) 
